cli/tools/test/reporters/junit.rs

```diff
@@ -40,13 +40,23 @@ impl JunitTestReporter {
 
 impl TestReporter for JunitTestReporter {
   fn report_register(&mut self, description: &TestDescription) {
-    self.cases.insert(
-      description.id,
-      quick_junit::TestCase::new(
-        description.name.clone(),
-        quick_junit::TestCaseStatus::skipped(),
-      ),
+    let mut case = quick_junit::TestCase::new(
+      description.name.clone(),
+      quick_junit::TestCaseStatus::skipped(),
     );
+    case.extra.insert(
+      String::from("filename"),
+      description.location.file_name.clone(),
```

Comment:
I also think we should strip out `file://` prefix from these. I haven't found an example of JUnit report that used file URL. That should be something like `.strip_prefix("file://")`